### PR TITLE
ImageService v2: Fixing Create Properties

### PR DESCRIPTION
**Pull request by jtopjian**
For #259

### CODE REVIEW

File: acceptance/openstack/imageservice/v2/images_test.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	"testing"
+
+	"github.com/gophercloud/gophercloud/acceptance/clients"
+	"github.com/gophercloud/gophercloud/acceptance/tools"
+)
+
+func TestImagesCreateDestroyEmptyImage(t *testing.T) {
+	client, err := clients.NewImageServiceV2Client()
+	if err != nil {
+		t.Fatalf("Unable to create an image service client: %v", err)
```
Comment (jtopjian) [inline acceptance/openstack/imageservice/v2/images_test.go]: as an aside: `%v` here was always a mistake. When I did the initial cleanup of the tests, I had meant this to be `%#v`, but without understanding the structure of the error. I had thought there would be good bit of info in the error structure to print out. Turns out `%#v` would have been a bad choice. `%v` could easily just be `%s`. Maybe I'll fix that globally one day.

**Comment by jtopjian** [comment]
@jrperritt ping for review

**Comment by jrperritt** [comment]
Is there a unit test for this?

**Comment by jtopjian** [comment]
Yes :)

**Comment by jtopjian** [comment]
Thank you!

**Pull request merged**